modules/bound-sa-tokens-configuring.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * authentication/bound-service-account-tokens.adoc
+
+[id="bound-sa-tokens-configuring_{context}"]
+= Configuring bound service account tokens using volume projection
+
+You can configure Pods to request bound service account tokens by using volume projection.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created a service account. This procedure assumes that the service account is named `build-robot`.
+
+.Procedure
+
+. Optionally, set the service account issuer.
++
+This step is typically not required if the bound tokens are used only within the cluster.
+
+.. Edit the `cluster` authentication object.
++
+----
+$ oc edit authentications cluster
+----
+
+.. Set the `spec.serviceAccountIssuer` field to the desired service account issuer value. The default is [x-]`https://kubernetes.default.svc`.
++
+[source,yaml]
+----
+spec:
+  serviceAccountIssuer: https://test.default.svc <1>
+----
+<1> The issuer value must start with `https://`.
```

Comment:
I think it would be more helpful for this message to state: 

```
This value should be a url from which the recipient of a bound token can source the public keys necessary to verify the signature of the token.
```